docs/ref/models/fields.txt

```diff
@@ -521,12 +521,22 @@ optional arguments:
     * For :class:`DateTimeField`: ``default=timezone.now`` - from
       :func:`django.utils.timezone.now`
 
+    .. versionchanged:: 1.10
+
+        Adding a field with an ``auto_now_add`` attribute after the
+        model has been created asks for a default value, providing
+        the default option of using the time the migration is
+        applied. The rationale is that the semantic of ``auto_now_add`` is
+        having a timestamp for the record **creation** while applying it's
+        just an update.
+
 The default form widget for this field is a
 :class:`~django.forms.TextInput`. The admin adds a JavaScript calendar,
 and a shortcut for "Today". Includes an additional ``invalid_date`` error
 message key.
 
-The options ``auto_now_add``, ``auto_now``, and ``default`` are mutually exclusive.
+The options ``auto_now_add``, ``auto_now``, and ``default`` are mutually
```

Comment:
Try to avoid making unrelated cosmetic cleanups in a patch like this because it confuses things and makes git blame less helpful. I'm just going to revert the changes to this file. The versionchanged annotation doesn't seem necessary to me.